metecho/api/models.py

```diff
@@ -288,6 +289,7 @@ class Project(
     repo_name = StringField()
     name = StringField(unique=True)
     description = MarkdownField(blank=True, property_suffix="_markdown")
+    has_truncated_issues = models.BooleanField(default=False)
```

Comment:
@jgerigmeyer @oluoluoxenfree I added a new boolean field `has_truncated_issues` to the Project. Should be able to tell us if the repo has more than 1000 issues